projects/archiver/src/tasks/notification/helpers/device-notifications-helpers.ts

```diff
@@ -31,12 +33,14 @@ const createScheduleNotificationEndpoint = (
 
 /**
  * TODO
- * it will work now only for Daily Editions in UK
+ * +3 hours from issue date is defaulted in.
+ * This will (obviously) work only for Daily Editions in UK
  * In the future we will need to make it more generic (for US and Australia)
  **/
-export const createScheduleTime = (issueDate: string): string => {
-    const THREE_AT_NIGHT = 'T03:00:00Z'
-    return `${issueDate}${THREE_AT_NIGHT}`
+export const createScheduleTime = (issueDate: string, offset = 3): string => {
+    const issueDateAsDate: Date = parse(issueDate, 'yyyy-MM-dd', new Date())
```

Comment:
This could be `moment.parse(issueDate, 'yyyy-MM-dd')` (although that pattern is standard for moment so the format string isn't needed)
Then you could call `issueDateAsDate.set('hours',3)` to set the time to 3am. I don't think adding a time to a Date created without a time is explicit in what it means.

Reply:
I don't want to set hours to 3 though - I want to apply an offset.

The offset for Australia will be -7.